operator/pkg/manifest/operations.go

```diff
@@ -159,29 +168,33 @@ func (o *Operations) Install(deployInfo InstallInfo) (bool, error) {
 		return false, err
 	}
 
-	targetResources, existingResources, err := o.getClusterResources(deployInfo, OperationCreate)
+	resourceLists, err := o.getClusterResources(deployInfo, OperationCreate)
 	if err != nil {
 		return false, err
 	}
 
-	if existingResources == nil && len(targetResources) > 0 {
-		if _, err = o.helmClient.PerformCreate(targetResources); err != nil {
+	if resourceLists.Installed == nil && len(resourceLists.Target) > 0 {
+		if _, err = o.helmClient.PerformCreate(resourceLists.Target); err != nil {
 			return false, err
 		}
 	} else {
-		if _, err = o.helmClient.PerformUpdate(existingResources, targetResources, true); err != nil {
+		if _, err = o.helmClient.PerformUpdate(resourceLists, true); err != nil {
 			return false, err
 		}
 	}
 
+	// verify namespace resource along with manifest resources
+	//nolint:gocritic
+	resourcesToBeVerified := append(resourceLists.Target, resourceLists.Namespace...)
```

Comment:
If the `resourceLists.Namespace` always has to append to `resourceLists.Target` to be used as `resourcesToBeVerified`, can we direct return `resourcesToBeVerified` from `getClusterResources` instead of `Namespace`?

Reply:
That was my first idea as well, but the namespace resource is implicit and doesn’t belong to the chart, so it only has to be waited for.